sct-commons/src/test/java/org/lfenergy/compas/sct/commons/dto/ResumedDataTemplateTest.java

```diff
@@ -217,25 +217,25 @@ private static Stream<Arguments> daParametersProviderNotUpdatable() {
     void findFirstValue_should_return_value(){
         // Given
         ResumedDataTemplate rDTT = new ResumedDataTemplate();
-        DaTypeName da1 = createDa("da1", TFCEnum.CF, true, Map.of(10L, "a value"));
+        DaTypeName da1 = createDa("da1", TFCEnum.CF, true, Map.of(10L, "a iedName"));
```

Comment:
I don't sure `a iedName` is a good value for testing, the value less likely be the name of an ied, it depends on the bType. 
  [TPredefinedBasicTypeEnum](https://github.com/com-pas/compas-scl-xsd/blob/12654b7586c7dde5068c739ce099492a82ee2356/src/main/resources/xsd/SCL2003/SCL_Enums.xsd#L401)